src/components/member-task-update/index.js

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Modal from '@components/UI/modal/index';
+import Spinner from '@components/UI/spinner';
+import { TASK_TYPE } from '@constants/AppConstants';
+import { useTaskContext } from '@store/tasks/tasks-context';
+import classNames from './member-task-update.module.scss';
+import { moveTask } from '../../helper-functions/action-handlers';
+
+const MemberTaskUpdate = () => {
+  const {
+    showMemberTaskUpdateModal,
+    setShowMemberTaskUpdateModal,
+    isNoteworthy,
+    taskId,
+  } = useTaskContext();
+
+  const [isUpdating, setIsUpdating] = useState(false);
+  const [updateStatus, setUpdateStatus] = useState('');
+
+  const changeTaskType = async (taskid) => {

```

Comment:
```suggestion
  const changeTaskType = async (taskId) => {

```

Reply:
can't do this, because there is another taskId in that scope, so yarn lint gives an error